llama_stack/providers/remote/inference/fireworks/test.py

```diff
@@ -0,0 +1,257 @@
+import json
```

Comment:
so I don't really think we want to encourage unittests for these "adapters" because
- the actual adapter code tends to be relatively uncomplicated
- there just needs to be a whole lot of mocking

as a result, running the unittest doesn't help a whole lot in giving guarantees about the stability.

I am curious to know what you think though.